### ISSUE TITLE
[BUG]  ZugferdDocumentBuilder `->setDocumentSummation()` `$taxTotalAmount` parametter is not optional

### ISSUE THREAD

**Issue by jameskitt616** [horstoeko/zugferd#113]
**Describe the bug**
When creating a new ZugferdDocumentBuilder and setting the Document Summation but only setting the required fields for the `setDocumentSummation()` method, the method returns a `Warning: Undefined array key 0` Exception due not setting the `$taxTotalAmount` parameter, which is marked nullable/optional.

**To Reproduce**
```
$document = ZugferdDocumentBuilder::CreateNew(ZugferdProfiles::PROFILE_EN16931);
...
$document->setDocumentSummation(123.45, 123.45);
```

**Expected behavior**

**Screenshots**

**Sample Code**
To fix the exception, the `$taxTotalAmount` paramter must be set
```
$document = ZugferdDocumentBuilder::CreateNew(ZugferdProfiles::PROFILE_EN16931);
...
$document->setDocumentSummation(123.45, 123.45, null, null, null, null, 0.0);
```

**Additional information:**
 - OS: Limux
 - OS-Version: Debian GNU/Linux 12
 - PHP-Version: PHP 8.3.11

**Additional context**
The description of the `$taxTotalAmount` parameters says:
> – BT-110/ 111, From MINIMUM/ BASIC WL **if BT--6 is not null $taxTotalAmount = BT--111.**

Which reads to me if `BT--6` (`$lineTotalAmount`) is not set (which is not in my example), this also does not need to be set.

**Comment by horstoeko** [comment]
Hi @jameskitt616,

Thank you very much for your issue. I will take care of your problem today (if I have time). I think I have already identified the problem.

Kind regards

**Comment by horstoeko** [comment]
Hi @jameskitt616,

I have fixed the problem with release [1.0.61](https://github.com/horstoeko/zugferd/releases/tag/v1.0.61). If this has solved your problem, please close the issue yourself.

Kind regards

**Comment by jameskitt616** [comment]
Thank's a lot! That fixed it.
